fourier.c

```diff
@@ -45,42 +45,65 @@
 #define VERSION "0.4.5"
 #endif
 
+#if (GIMP_MAJOR_VERSION == 3) || ((GIMP_MAJOR_VERSION == 2) && (GIMP_MINOR_VERSION >= 99))
+#ifdef HAVE_GETTEXT
+#include <libintl.h>
+#define d_(String) String
+#ifdef gettext_noop
+#    define N_(String) gettext_noop (String)
+#else
+#    define N_(String) (String)
+#endif
+#define _(String) gettext (String)
+#else
+/* No i18n for now */
+#define N_(x) x
+#define d_(x) x
+#define _(x) x
+#endif
+#else
+/* no gettext used with gimp2 version */
+#define N_(x) x
+#define d_(x) x
+#define _(x) x
+#endif
+
 /** Defines ******************************************************************/
 
 #define PLUG_IN_BINARY "fourier"
 #define PLUG_IN_NAME "plug_in_fft"
 #define PLUG_IN_VERSION "Jun 2024, " VERSION
 
-#define PLUG_IN_AUTHOR "Remi Peyronnet"
-#define PLUG_IN_MENU_LOCATION "<Image>/Filters/Generic"
+static char *PLUG_IN_AUTHOR = "Remi Peyronnet";
+static char *PLUG_IN_MENU_LOCATION = d_("<Image>/Filters/Generic");
 
-#define PLUG_IN_PROC "plug-in-fourier"
+static char *PLUG_IN_PROC = d_("plug-in-fourier");
 
-#define PLUG_IN_DIR_PROC "plug-in-fourier-forward"
-#define PLUG_IN_DIR_MENU_LABEL "FFT Forward"
-#define PLUG_IN_DIR_SHORT_DESC "This plug-in applies a FFT to the image, for educationnal or effects purpose."
-#define PLUG_IN_DIR_DESC  "Apply an FFT to the image. This can remove (for example) moire patterns from images scanned from books:\n\n" \
-                          "    The image should be RGB (Image|Mode|RGB)\n\n" \
-                          "    Remove the alpha layer, if present (Image|Flatten Image)\n\n" \
-                          "    Select Filters|Generic|FFT Forward\n\n" \
-                          "    Use the preselected neutral grey to effectively remove any moir patterns from the image. Either paint over any patterns or\n\n" \
-                          "     - In the Layers window, select the layer, and 'Duplicate Layer'\n" \
-                          "     - Select Colours|Brightness-Contrast. Increase the Contrast to see any patterns.\n" \
-                          "     - Use the Rectangular and/or Elliptical Selection tools to select any patterns on the contrast layer.\n" \
-                          "     - Then remove the contrast layer leaving the original FFT layer with the selections.\n" \
-                          "     - Then select Edit|Fill with FG colour, remembering to cancel the Selection afterwards!\n\n" \
-                          "    Select Filters|Generic|FFT Inverse\n\n" \
-                          "Voila, an image without the moire pattern!"
+static char *PLUG_IN_DIR_PROC = d_("plug-in-fourier-forward");
```

Comment:
I was doing a compare with gimp2, and added that. Looking at the help-plugin-browser-info, seems those stay the same. Seems it can come out. Consider it a copy-n-paste error.